modules/exploits/windows/browser/ms14_064_ole_code_execution.rb

```diff
@@ -11,17 +11,17 @@ class Metasploit4 < Msf::Exploit::Remote
   Rank = ExcellentRanking
 
   include Msf::Exploit::Remote::BrowserExploitServer
-  include Msf::Exploit::Remote::BrowserAutopwn
+  #include Msf::Exploit::Remote::BrowserAutopwn
   include Msf::Exploit::Powershell
 
-  autopwn_info({
-    :ua_name    => HttpClients::IE,
-    :ua_minver  => "3.0",
-    :ua_maxver  => "10.0",
-    :javascript => true,
-    :os_name    => OperatingSystems::Match::WINDOWS,
-    :rank       => ExcellentRanking
-  })
+  #autopwn_info({
```

Comment:
We can. I looked at other revoked browserautopwn modules and they're all the same way so I decided to go with the flow. Let me know if the workaround actually works and then I'll delete the lines.